src/devices/inovelli.ts

```diff
@@ -484,13 +491,13 @@ const inovelliExtend = {
         return {
             fromZigbee,
             toZigbee: [
+                tz.on_off,
```

Comment:
@InovelliUSA found the problem. Apparently order matters when setting up the toZigbee converters. Putting `tz.on_off` first before `light_onoff_brightness` solves it.